repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -52,7 +52,15 @@ object ProjectUpdateRequest {
   implicit val updateRequestFormat: OFormat[ProjectUpdateRequest] = Json.format[ProjectUpdateRequest]
 }
 
-final case class Version(value: String) extends AnyVal
+final case class Version(name: String, message: String, target: String, commit: Commit)
+object Version {
+  implicit val versionPlayFormat: OFormat[Version] = Json.format[Version]
```

Comment:
Why you named it versionPlayFormat, not versionFormat, like you did below with commitFormat?